ObserverSet.swift

```diff
@@ -26,108 +26,122 @@
 //
 
 import Dispatch
-import Foundation
 
-public class ObserverSetEntry<Parameters> {
+/// A reference to an entry in the list of observers. Use this to remove an observer.
+open class ObserverSetEntry<Parameters> {
     
-    fileprivate weak var object: AnyObject?
-    fileprivate let operationQueue: OperationQueue?
-    fileprivate let f: (AnyObject) -> (Parameters) -> Void
+    public typealias Callback = (Any) -> (Parameters) -> Void
     
-    fileprivate init(object: AnyObject, operationQueue: OperationQueue?, f: @escaping (AnyObject) -> (Parameters) -> Void) {
-        self.object = object
-        self.operationQueue = operationQueue
-        self.f = f
+    fileprivate weak var observer: AnyObject?
+    fileprivate let callBack: Callback
```

Comment:
```suggestion
    fileprivate let callback: Callback
```

Because "callback" is a single word, I would lowercase the "b" throughout this PR.